js/migrate_issues.js

```diff
@@ -3,153 +3,146 @@ import fs from 'fs'
-    const issuesFile = 'issues.json'
-    let repos = fs.existsSync(issuesFile)
-        ? JSON.parse(fs.readFileSync('issues.json', 'utf-8'))
-        : await getTargetIssues(previousOwner, targetLabel, REPOS)
+  const issuesFile = 'issues.json'
+  let repos = fs.existsSync(issuesFile)
+    ? JSON.parse(fs.readFileSync('issues.json', 'utf-8'))
+    : await getTargetIssues(previousOwner, targetLabel, REPOS)
 
-    repos.forEach((repo) => {
-        repo.issues.forEach(async (issue) => {
-            const issueMeta = `${issue.migrationNotice}\n\`\`\`\n${[
-                issue.author,
-                issue.date,
-                issue.labels,
-            ].join('')}\`\`\``
-            const commentsToAdd = issue.comments
-                ? `\n--------\r\n### Original Comments: \n\n${issue.comments.join(
-                      '\n\n'
-                  )}`
-                : ''
-            try {
-                await octokit.rest.issues.create({
-                    owner: newOwner,
-                    repo: repo.a8cName,
-                    title: issue.title,
-                    body: [issueMeta, issue.body].join('\r\n') + commentsToAdd,
-                })
-            } catch (e) {
-                console.error(
-                    `Could not create an issue "${issue.title}" in ${repo.a8cName} due to ${e}`
-                )
-            }
+  repos.forEach(repo => {
+    repo.issues.forEach(async issue => {
+      const issueMeta = `${issue.migrationNotice}\n\`\`\`\n${[
+        issue.author,
+        issue.date,
+        issue.labels,
+      ].join('')}\`\`\``
+      const commentsToAdd = issue.comments
+        ? `\n--------\r\n### Original Comments: \n\n${issue.comments.join(
+            '\n\n'
+          )}`
+        : ''
+      try {
+        await octokit.rest.issues.create({
+          owner: newOwner,
+          repo: repo.a8cName,
+          title: issue.title,
+          body: [issueMeta, issue.body].join('\r\n') + commentsToAdd,
         })
+      } catch (e) {
+        console.error(
+          `Could not create an issue "${issue.title}" in ${repo.a8cName} due to ${e}`
+        )
+      }
     })
+  })
 }
 
 const REPOS = [
-    {
-        name: 'cccatalog-frontend',
-        fullName: 'CC Search Frontend repository',
-        a8cName: 'ccsearch-frontend',
-    },
-    {
-        name: 'cccatalog-api',
-        fullName: 'CC Search API repository',
-        a8cName: 'ccsearch-api',
-    },
-    {
-        name: 'cccatalog',
-        fullName: 'CC Search Catalog repository',
-        a8cName: 'ccsearch-catalog',
-    },
+  {
+    name: 'cccatalog-frontend',
+    fullName: 'CC Search Frontend repository',
+    a8cName: 'ccsearch-frontend',
+  },
+  {
+    name: 'cccatalog-api',
+    fullName: 'CC Search API repository',
+    a8cName: 'ccsearch-api',
+  },
+  {
+    name: 'cccatalog',
+    fullName: 'CC Search Catalog repository',
+    a8cName: 'ccsearch-catalog',
+  },
```

Comment:
Can read [`data/github.yml`](https://github.com/WordPress/openverse/blob/main/data/github.yml) instead of hardcoding them here.